app/frontend/src/assets/scss/style.scss

```diff
@@ -57,3 +56,7 @@ body > .v-content {
   border: 2px solid #003366;
   color: #003366 !important;
 }
+
```

Comment:
Not sure if this is the PR or place to mention this, the button sizes and the text placement seems inconsistent - Login/Logout is centred nicely and has a nice size.  The Back/Next buttons in the wizard have the text too low and the buttons feel like they aren't as tall.  Not sure what we are doing for height/width standards, but the text should always be centred vertically and horizontally.  This was on a chrome browser.

The Get Token for DEV/TEST buttons are ok, I think we need to fix the BCGovPrimary or just lose it.

BC-Gov-PrimaryButton
padding: 12px 32px!important; // remove

BC-Gov-PrimaryButton.light.v-btn
border: 2px solid #036; // this should be 1px more consistent with .v-btn--outlined

Maybe only use our own styles for colour?

Reply:
Yes all good points. Text alignment for buttons already captured in https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-882
Will add the other notes to that subtask. You're right that the BC-Gov-PrimaryButton styling doesn't quite match the mockups.